weave-js/src/common/components/elements/ModifiedDropdown.tsx

```diff
@@ -58,6 +58,14 @@ const simpleSearch = (options: DropdownItemProps[], query: string) => {
     .value();
 };
 
+const getOptionProps = (opt: Option, hideText: boolean) => {
+  const {text, ...props} = opt;
+  props.text = hideText ? null : (
+    <OptionWithTooltip text={opt.text as string} />
+  );
+  return props;
```

Comment:
Adding the option to hide the text prop so that we can still search it's value, but only display the JSX label if we'd like